bundle/android/strings.xml

```diff
@@ -311,7 +311,7 @@ translators: Block name. %s: The localized block name -->
     <!-- translators: %s: Select font size option value e.g: "Selected: Large".
 translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="gutenberg_native_selected_s" tools:ignore="UnusedResources">Selected: %s</string>
-    <string name="gutenberg_native_set_as_featured_image" tools:ignore="UnusedResources">Set as Featured Image </string>
+    <string name="gutenberg_native_set_as_featured_images" tools:ignore="UnusedResources">Set as Featured Images </string>
```

Comment:
I couldn't find a commit where this string was changed, do you have any clue why it changed?

Reply:
good catch, I _think_ I might have changed this and the other change above while trying to figure out the issue.

Reply:
I know what I did 🤦 , I ran the `i18n:update` command while on the wrong gutenberg ref.

Reply:
Oh, that explains the changes 😅.

Reply:
Fixed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4464/commits/5821350c3c9bd9d127fe04e33fba2115c61bccdf